### DISCUSSION TITLE
Error when trying to add a track to a playlist

### DISCUSSION THREAD

**Discussion by Official404** [JohnnyCrazy/SpotifyAPI-NET#816] [Q&A]
Hello!
I don't know if im doing anything wrong but when i try to add a track to a playlist i only get back

```TEXT
System.AggregateException: One or more errors occurred. (Invalid playlist Id)
 ---> SpotifyAPI.Web.APIException: Invalid playlist Id
   at SpotifyAPI.Web.Http.APIConnector.ProcessErrors(IResponse response)
   at SpotifyAPI.Web.Http.APIConnector.DoRequest(IRequest request)
   at SpotifyAPI.Web.Http.APIConnector.DoSerializedRequest[T](IRequest request)
   at SpotifyAPI.Web.Http.APIConnector.SendAPIRequest[T](Uri uri, HttpMethod method, IDictionary`2 parameters, Object body, IDictionary`2 headers)
```

Curtrently my code looks like this

```C#
public void AddTrackToPlaylist(string playlist, string trackid)
{
    try {
        
        var request = new PlaylistAddItemsRequest(new List<string> { trackid });
        _SpotifyClient.Playlists.AddItems(playlist, request);
    }
    catch (APIException e) {
        Console.WriteLine(e);
    }
}
```
And the playlist variable is "0rvwvhgGqs54s3P2xHWgcz" and the trackid variable is "7g2zJC8JmpZF2vcXjZOHOe"

What ever i do it dosent seem to work so now im thinking that im doing something wrong. Thanks in advance if anyone could figure this out.

**Comment by JohnnyCrazy** [comment]
You might want to check if you actually own the playlist. You can only add tracks to the current authenticated user's playlist or collaborative ones.

Feel free to re-open if not solved.
